pkg/rtc/room.go

```diff
@@ -277,21 +277,23 @@ func (r *Room) Join(participant types.LocalParticipant, opts *ParticipantOptions
 		go func() {
 			// when a participant subscribes to another participant,
 			// send speaker update if the subscribed to participant is active.
-			speakers := r.GetActiveSpeakers()
-			for _, speaker := range speakers {
-				if livekit.ParticipantID(speaker.Sid) == publisherID {
-					_ = p.SendSpeakerUpdate(speakers)
-					break
+			/*
+				speakers := r.GetActiveSpeakers()
```

Comment:
I don't know if these changes are necessary, might try reverting them to see what impact they have. I commented these out when I was debugging something completely different.